src/commands/build.ts

```diff
@@ -131,10 +168,60 @@ export default class Build extends BaseCommand {
       );
     }
 
+    if (flags.path) {
+      let trackingPlans = [] 
+      for (const plan of configPlans) {
+        const oldPath = './' + plan.path
+        const configPlan = await loadTrackingPlan(this.configPath, plan)
+        const { path } = await this.prompt({
+          type: "input",
+          name: "path",
+          message: this.formatQuestion(
+            `Enter a directory for the Tracking Plan ${chalk.green(
+              configPlan?.name
+            )} output:`,
+            [
+              "The generated client will be stored in this directory.",
+              "Directories will be automatically created, if needed.",
+            ]
+          ),
+          filter: normalize,
+          default: plan?.path
+        });
+
+        plan!.path = path;
+
+        // fetch tracking plan and store in new path
+        let newTrackingPlan: SegmentAPI.TrackingPlan | undefined;
+        try {
+          CliUx.ux.action.status = `Fetching tracking plan ${plan.name}.`;
+          newTrackingPlan = await fetchTrackingPlan(
+            plan.id,
+            flags.token ?? this.apiToken!
+          );
+        } catch (error) {
+          const errorMessage = isWrappedError(error)
+            ? error.description
+            : "API request failed";
+          debug(`${errorMessage}. Using local copy of ${plan.name} instead.`);
+          CliUx.ux.action.status = `Error fetching tracking plan ${plan.name}, using local copy instead.`;
+        }
+        if (newTrackingPlan !== undefined) {
+          CliUx.ux.action.status = `Updating local copy of tracking plan ${plan.name}.`;
+          // Update plan.json with the latest Tracking Plan.
+          await writeTrackingPlan(this.configPath, newTrackingPlan, plan);
+          CliUx.ux.action.status = `Plan ${plan.name} updated.`;
+        }
+        trackingPlans.push({name: configPlan?.name, path: plan.path, id: plan.id})
+        this.deleteFolder(oldPath)
```

Comment:
I don't think we should delete folders. We should assume the user will delete things if their previous configuration was incorrect. This might cause some bad issues if people have things in that folder which is not that uncommon (we actually do that in TW itself).